jsapp/js/editorMixins/cascadeMixin.es6

```diff
@@ -49,7 +49,7 @@ export const cascadeMixin = {
         choices: inp
       });
       if (tmpSurvey.choices.length === 0) {
-        throw new Error(t('Paste your formatted table from excel in the box below.'));
+        throw new Error('Paste your formatted table from excel in the box below.');
```

Comment:
@magicznyleszek doesn't this message get shown to the user?

https://github.com/kobotoolbox/kpi/blob/a9a24018d321fad4a83108c51b79fa2d38d263c1/jsapp/js/editorMixins/cascadeMixin.es6#L71-L85